MangoPay/PayIn.php

```diff
@@ -13,6 +13,23 @@ class PayIn extends Transaction
      */
     public $CreditedWalletId;
 
+    /**
+     * The mode applied for the 3DS protocol for CB, Visa, and Mastercard.
+     * Default values: DEFAULT. Allowed values: DEFAULT, FORCE, NO_CHOICE
+     *
+     * @var string
+     */
+    public $SecureMode;
+
+    /**
+     * The 3DS protocol version to be applied to the transaction. Allowing values: V1, V2_1
+     *
+     * @see https://mangopay.com/docs/concepts/payments/payment-methods/card/3ds
+     * @see https://mangopay.com/docs/endpoints/direct-card-payins#direct-card-payin-object
+     * @var string
+     */
+    public $Requested3DSVersion;
```

Comment:
What is the reason for adding this to the PayIn class?

Reply:
This property was identified in both the documentation and PHP SDK. I verified its existence through debug logs and implemented it accordingly.